picard/coverart/providers/caa.py

```diff
@@ -103,8 +108,13 @@ def caa_url_fallback_list(desired_size, thumbnails):
     for item_id, item in reversed_map.items():
         if item_id == -1 or item_id > desired_size:
             continue
-        if item.thumbnail in thumbnails:
-            urls.append(thumbnails[item.thumbnail])
+        url = thumbnails.get(item.thumbnail, None)
+        if url is None:
+            size_alias = _CAA_THUMBNAIL_SIZE_ALIASES.get(item.thumbnail, None)
+            if size_alias is not None:
+                url = thumbnails.get(size_alias, None)
```

Comment:
Does it actually happen that we have `"large"` in the response but not `"500"`? Because as I understand it this is the use case here.

Reply:
Yes, that's possible :(
This is why I want to add "compatibility" with old ("large" & "small") and new ("250", "500"), even newer ("1200") cases, because `index.json` files aren't always properly updated.

https://ia802706.us.archive.org/18/items/mbid-ad7a42c2-8baa-4f6c-8d71-1ae2f7cf3afa/index.json

```json
{
  "images": [
    {
      "types": [
        "Front"
      ],
      "front": true,
      "back": false,
      "edit": 19432737,
      "image": "http://coverartarchive.org/release/ad7a42c2-8baa-4f6c-8d71-1ae2f7cf3afa/2366957454.jpg",
      "comment": "",
      "approved": true,
      "thumbnails": {
        "large": "http://coverartarchive.org/release/ad7a42c2-8baa-4f6c-8d71-1ae2f7cf3afa/2366957454-500.jpg",
        "small": "http://coverartarchive.org/release/ad7a42c2-8baa-4f6c-8d71-1ae2f7cf3afa/2366957454-250.jpg"
      },
      "id": "2366957454"
    }
  ],
  "release": "http://musicbrainz.org/release/ad7a42c2-8baa-4f6c-8d71-1ae2f7cf3afa"
}
```